All ingredient js iifes complete. Prices get added when selecting top…

wocaldwell opened the pull request:
All ingredient js iifes complete. Prices get added when selecting toppings. Price is logged in console

redeemefy commented:
- In Cheeze, Condiment, Bread, Protein, you aren't using the function argument. Also, the main object variable is different on each object.
- I should be able to do the following in the console... `SandwichMaker.addProtein['tofu']` and the console should return `1.99`.

wocaldwell commented:
Fixed object names in js files. Corrected function argument naming. However, when typing `SandwichMaker.addProtein['tofu']` console returns `undefined`. But if I enter `SandwichMaker.addProtein('tofu')` it returns `1.99`. Is this correct?
